sdk/textanalytics/ai-text-analytics/src/detectLanguageResult.ts

```diff
@@ -32,12 +37,13 @@ export type DetectLanguageErrorResult = TextAnalyticsErrorResult;
 
 export function makeDetectLanguageResult(
   id: string,
-  detectedLanguages: DetectedLanguage[],
+  detectedLanguage: DetectedLanguage,
+  warnings: TextAnalyticsWarning[],
   statistics?: TextDocumentStatistics
 ): DetectLanguageSuccessResult {
   return {
-    ...makeTextAnalyticsSuccessResult(id, statistics),
-    primaryLanguage: primaryLanguage(detectedLanguages)
+    ...makeTextAnalyticsSuccessResult(id, warnings, statistics),
+    primaryLanguage: detectedLanguage
```

Comment:
should this be detectedLanguage instead of primaryLanguage now?

Reply:
I think the plan is to roll with primaryLanguage to leave the door open to expanding to multiple languages in the future without needing to change this property name.